common/djangoapps/enrollment/views.py

```diff
@@ -76,10 +78,29 @@ def has_api_key_permissions(self, request):
 
 class EnrollmentUserThrottle(UserRateThrottle, ApiKeyPermissionMixIn):
     """Limit the number of requests users can make to the enrollment API."""
+    # TODO: After confirming that reducing the throttle is successful, remove
+    # and clean up waffles. The rate limit has been increased over the course
+    # of a few months to account for unnecessary calls from the ecommerce
+    # service. These calls are no longer made and the plan is to set the
+    # rate limit back to its original state. LEARNER-5148
+
+    # Current rate limit
     THROTTLE_RATES = {
         'user': '40/minute',
+        'staff': '2000/minute',
     }
 
+    # Less aggressive reduction in throttle limit. This should not be necessary
+    if REDUCE_RATE_LIMIT_FOR_STAFF_FOR_ENROLLMENT_API.is_enabled():
+        THROTTLE_RATES = {
+            'user': '40/minute',
+            'staff': '400/minute',
+        }
+
+    # Original rate Limit before rate limit increases.
+    if USE_UNIVERSAL_RATE_LIMIT_FOR_ENROLLMENT_API.is_enabled():
+        rate = '40/minute'
```

Comment:
I incorrectly set throttle rate here to 
```
        THROTTLE_RATES = {
            'user': '40/minute',
        }
```

https://github.com/edx/edx-platform/commit/e35b35f6c2262c95df68858c92fec9d5f6a2f5ca#diff-a4a6f81c03f035ebb4441f342e27c62eL71

Reply:
Additionally, I didn't type the waffle flag correctly when enabling it testing.